open-metadata-implementation/view-services/subject-area-view/subject-area-view-server/src/main/java/org/odpi/openmetadata/viewservices/subjectarea/auditlog/SubjectAreaViewAuditCode.java

```diff
@@ -0,0 +1,168 @@
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.text.MessageFormat;
+import java.util.Arrays;
+
+/**
+ * The SubjectAreaAuditCode is used to define the message content for the OMRS Audit Log.
+ *
+ * The 5 fields in the enum are:
+ * <ul>
+ *     <li>Log Message Id - to uniquely identify the message</li>
+ *     <li>Severity - is this an event, decision, action, error or exception</li>
+ *     <li>Log Message Text - includes placeholder to allow additional values to be captured</li>
+ *     <li>Additional Information - further parameters and data relating to the audit message (optional)</li>
+ *     <li>SystemAction - describes the result of the situation</li>
+ *     <li>UserAction - describes how a user should correct the situation</li>
+ * </ul>
+ */
+
+
+//TODO the adding of terms for audit will need to pass the guid as a parameter and maybe the fqname. I do not see how I can add parameters in the beloe structure.
+
+public enum SubjectAreaViewAuditCode {
+    SERVICE_INITIALIZING("OMVS-SUBJECT-AREA-0001",
+            OMRSAuditLogRecordSeverity.STARTUP,
+            "The Subject Area View Service (OMVS) is initializing",
+            "The local server has started up the Subject Area OMVS.",
+            "No action is required. This is part of the normal operation of the Subject Area OMVS."),
+
+    SERVICE_INITIALIZED("OMVS-SUBJECT-AREA-0003",
+            OMRSAuditLogRecordSeverity.STARTUP,
+            "The Subject Area View Service (OMVS) is initialized",
+            "The Subject Area OMVS has completed initialization.",
+            "No action is required. This is part of the normal operation of the Subject Area OMVS."),
+
+    SERVICE_SHUTDOWN("OMVS-SUBJECT-AREA-0004",
+            OMRSAuditLogRecordSeverity.SHUTDOWN,
+            "The Subject Area View Service (OMVS) is shutting down",
+            "The local server has requested shut down of the Subject Area OMVS.",
+            "No action is required. This is part of the normal operation of the Subject Area OMVS."),
+
+    SERVICE_INSTANCE_FAILURE("OMVS-SUBJECT-AREA-0032",
+            OMRSAuditLogRecordSeverity.EXCEPTION,
+            "The Subject Area View Service (OMVS) is unable to initialize a new instance; error message is {0}",
+            "The view service detected an error during the start up of a specific server instance.  Its services are not available for the server.",
+            "Review the error message and any other reported failures to determine the cause of the problem.  Once this is resolved, restart the server."),
+    SERVICE_TERMINATING("OMVS-SUBJECT-AREA-0033",
+            OMRSAuditLogRecordSeverity.SHUTDOWN,
+            "The Subject Area View Service (OMVS) is shutting down server instance {0}",
+            "The local handlers has requested shut down of the Subject Area OMVS.",
+            "No action is required. This is part of the normal operation of the service."),
```

Comment:
Need to add that operato shouldverify that shutdown was intended.

Reply:
added